core/jobs/src/test/java/com/cognifide/aet/job/common/modifiers/login/LoginModifierConfigTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.job.common.modifiers.login;
+
+import static com.cognifide.aet.job.common.modifiers.login.LoginModifierConfig.DEFAULT_LOGIN;
+import static com.cognifide.aet.job.common.modifiers.login.LoginModifierConfig.DEFAULT_PASSWORD;
+import static com.cognifide.aet.job.common.modifiers.login.LoginModifierConfig.LOGIN_PAGE_PARAM;
+import static com.cognifide.aet.job.common.modifiers.login.LoginModifierConfig.LOGIN_PARAM;
+import static com.cognifide.aet.job.common.modifiers.login.LoginModifierConfig.PASSWORD_PARAM;
+import static org.junit.Assert.assertEquals;
+
+import com.cognifide.aet.job.api.exceptions.ParametersException;
+import com.googlecode.zohhak.api.TestWith;
+import com.googlecode.zohhak.api.runners.ZohhakRunner;
```

Comment:
It's a completely separate conversation but, out of curiosity, have you considered migrating to JUnit 5?

Reply:
Sure, generally this is good idea to migrate to JUnit 5 - but there was never time for that.
This is definitely the thing you could help with and create a PR 😃